_docs/_latest/configure/general/non-org-root.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Running Forseti from Non-Organization Root
+order: 100
+---
+
+# {{ page.title }}
+
+By default, Forseti is designed to be installed with complete
+organization access, and run with the organization as the root node in the
+resource hierarchy.
+
+But, you also have the option to run Forseti on a subset of resources:
+1. if you are Org Admin, and you want to run Forseti on a specific folder
+1. if you are Folder Admin, and you want to run Forseti on a specific folder
+1. if you are Project Admin, and you want to run Forseti on projects
+that are only owned by you
+
+Inventory, Data Model, and Scanner will be supported for use on these subset
+of resources, but Explain will not be supported.
+
+## How to Install
+
+Run the Forseti [Installer]({% link _docs/latest/setup/install.md %}).
+
+By default, the installer will try to assign org-level roles. If you are not
+Org Admin, there will be errors, but you can safely disregard, as you will
+assign the correct roles later.
```

Comment:
Should we make it very specific that user will need to assign the roles manually?